markdown/en/ff-sortbox.md

```diff
@@ -1,24 +1,27 @@
 ## Overview
-The simplest way of using a sortbox, is to simply use the default settings.
-This will display all sort options defined in the FACT-Finder backend as a dropdown.
-```html
-<ff-sortbox></ff-sortbox>
-```
+The sortbox lets a user sort products by the criteria defined in the FACT-Finder backend. We provide two different
+kinds of sortboxes you can choose from. The `ff-sortbox-select` is a native HTML `select` element.
+The `ff-sortbox` is a custom build dropdown element. Both can be used out of the box or customized to your special needs.
+
+## Native HTML select
+The code below will display a native HTML `select`. The template specified through the `data-template` attribute is optional.
+It will default to `<option>{{description}}</option>` if not present.
```

Comment:
```markdown
You can choose from two different kinds of sortboxes. The `ff-sortbox-select` is a native HTML `select` element. 
The `ff-sortbox` is a custom-built dropdown element. Both can be used with or without customization.

## Native HTML select
The code below will display a native HTML `select`. The template specified through the `data-template` attribute is 
optional. It will default to `<option>{{description}}</option>` if not present.
```